### ISSUE TITLE
Dataset size

### ISSUE THREAD

**Issue by miltondts** [guilledufort/RENANO#2]
Hi, 
I've downloaded the _yst_ using the script:
https://github.com/guilledufort/RENANO/blob/master/datasets/yst/download_script.sh

Yet the size is not the 8GB mentioned in the README.md, but rather it is 6,485,289,396 bytes (6.5GB):

```
-rw-r--r-- 1 root root  877290596 Apr 27 22:22 ERR1539081.fastq
-rw-r--r-- 1 root root  170684094 Apr 27 22:24 ERR1539082.fastq
-rw-r--r-- 1 root root  882744210 Apr 27 22:30 ERR1539083.fastq
-rw-r--r-- 1 root root 3547181562 Apr 27 22:57 ERR1539084.fastq
-rw-r--r-- 1 root root 1007388934 Apr 27 23:02 ERR1539085.fastq
```

Is the information in the README.md incorrect?

**Comment by guilledufort** [comment]
Hi, thank you for reporting this issue.

The reported sizes of datasets _yst_ and _sor_ were incorrect indeed. The size you report (6,485,289,396 bytes) is the correct one for dataset _yst_. For dataset _sor_ the correct size is 133,259,667,138 bytes (133GB). The problem originated from a series of result files generated with an older version of the software that contained a bug. This has now been fixed in the README and will be corrected in the full documentation of the software. It is important to note that the compression results reported in the documentation should not see any changes.

Moreover, checking this issue I found that at the last minute I introduced a critical bug in the software uploaded to GitHub (when fixing compilation warnings) that has now been fixed. I recommend you download the latest version from GitHub. The version in Bioconda will be updated shortly.
